### ISSUE TITLE
Some bugs of the new version.

### ISSUE THREAD

**Issue by LWShowTime** [dvlab-research/LISA#54]
LISA.py 72行，kwargs没有"train_mask_decoder"这个key。后面的out_dim什么也都缺。
![image](https://github.com/dvlab-research/LISA/assets/51390653/eb36a4f7-ba16-4bbd-9017-481ed4075080)

**Comment by X-Lai** [comment]
Did you use the `v0` version model?

For the new version, please use the `v1` version model. The keys you mentioned already exist in the config of `v1` models: https://huggingface.co/xinlai/LISA-13B-llama2-v1/blob/main/config.json.

**Comment by LWShowTime** [comment]
@X-Lai The new version fix this problem. However, the mm_vision_tower is from pretrained path, so I still need to manually config it by directly writing my local path of CLIP in the code.

**Comment by cht619** [comment]
> @X-Lai The new version fix this problem. However, the mm_vision_tower is from pretrained path, so I still need to manually config it by directly writing my local path of CLIP in the code.

 请问怎么手动配置CLIP的路径？本地加载clip

**Comment by LWShowTime** [comment]
> > @X-Lai The new version fix this problem. However, the mm_vision_tower is from pretrained path, so I still need to manually config it by directly writing my local path of CLIP in the code.
> 
> 请问怎么手动配置CLIP的路径？本地加载clip

Check the load function of CLIP, the name can be used as the local path.

**Comment by DanielSHKao** [comment]
> Did you use the `v0` version model?
> 
> For the new version, please use the `v1` version model. The keys you mentioned already exist in the config of `v1` models: https://huggingface.co/xinlai/LISA-13B-llama2-v1/blob/main/config.json.

Hi, 

I'm wondering how I can use the v0 model. This is because I want to evaluate the results on validation set using chat.py. Do you provide a code to load v0 model?
